### PR TITLE
Feature/2014 add region admin

### PR DESCRIPTION
**Pull request by okedo**

### CODE REVIEW

File: src/app/shell/admin-tools/data/admins/create-admin/create-admin.component.ts [hunk 1]
```diff
@@ -40,14 +47,18 @@ const defaultValidators: ValidatorFn[] = [
...
   readonly phonePrefix = Constants.PHONE_PREFIX;
   readonly mailFormPlaceholder = Constants.MAIL_FORMAT_PLACEHOLDER;
   readonly adminsRole = AdminRoles;
 
   @Select(MetaDataState.institutions)
   institutions$: Observable<Institution[]>;
-  @Select(AdminState.selectedMinistryAdmin)
-  selectedMinistryAdmin$: Observable<MinistryAdmin>;
+  @Select(AdminState.selectedAdmin)
+  selectedAdmin$: Observable<BaseAdmin>;
```
Comment (humanmantis) [inline src/app/shell/admin-tools/data/admins/create-admin/create-admin.component.ts:59]: It would also be nice to add access modifiers to class fields

**Review by humanmantis — CHANGES_REQUESTED** [review]
I like this solution.

But a small note about access modifiers and types for class fields, it is better to always add them in new files and when working with old ones.

File: src/app/shell/details/provider-details/provider-details.component.spec.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ import { Workshop } from '../../../shared/models/workshop.model';
 import { ProviderDetailsComponent } from './provider-details.component';
 import { TranslateModule } from '@ngx-translate/core';
 
-describe('ProviderDetailsComponent', () => {
+fdescribe('ProviderDetailsComponent', () => {
```
[the remark this answers is not in the document]
Comment (okedo) [reply src/app/shell/details/provider-details/provider-details.component.spec.ts]: fixed

File: src/app/shell/admin-tools/data/admins/create-admin/create-admin.component.ts (hunk 1 above)
Comment (okedo) [reply src/app/shell/admin-tools/data/admins/create-admin/create-admin.component.ts]: added

**Review by humanmantis — APPROVED** [review]

**Pull request merged**